plat/rockchip/common/drivers/pmu/pmu_com.h

```diff
@@ -31,7 +31,7 @@
  * Use this macro to instantiate lock before it is used in below
  * rockchip_pd_lock_xxx() macros
  */
-DEFINE_BAKERY_LOCK(rockchip_pd_lock);
+static DEFINE_BAKERY_LOCK(rockchip_pd_lock);
```

Comment:
minor: It's a bit strange that this is being declared in the header at all. Better to move this and the functions below into pmu.c, and use `DECLARE_BAKERY_LOCK` here instead.